golem_messages/message/tasks.py

```diff
@@ -483,6 +483,19 @@ class SubtaskResultsAccepted(TaskMessage):
         'report_computed_task': base.MessageSlot(ReportComputedTask),
     }
 
+    @classmethod
+    def deserialize_with_header(cls, header, data, *args, **kwargs):  # noqa pylint: disable=arguments-differ
+        instance: SubtaskResultsAccepted = super().deserialize_with_header(
+            header, data, *args, **kwargs
+        )
+        instance.is_valid()
```

Comment:
I actually think this should be part of `TaskMessage`'s `deserialize_with_header` (given the fact that `TaskMessage` contains the base implementation of `is_valid`)

Reply:
done